kuma/wiki/forms.py

```diff
@@ -466,15 +467,25 @@ def akismet_parameters(self):
         for a model instance in
         ``RevisionAkismetSubmissionAdminForm.akismet_parameters`` method!
         """
-        language = self.cleaned_data.get('locale',
-                                         settings.WIKI_DEFAULT_LANGUAGE)
+        if self.instance and self.instance.pk and self.instance.document:
+            language = self.instance.document.locale
+        else:
+            language = settings.WIKI_DEFAULT_LANGUAGE
+
```

Comment:
This code isn't going to work in the translate view from what I'm reading. The reason being that the view doesn't pass the instance in to the form, which means there is no document to pluck the locale from. When creating a new translation the locale comes from the `tolocale` query param which gets passed into the form's POST data.

I'd propose that we do multiple checks here with the settings as the final fallback. For example: get locale from the instance provided, if no instance get locale from the form data, (maybe another check?), fallback.